keyboards/k_pro_v1_tkl/info.json

```diff
@@ -0,0 +1,125 @@
+{
+    "manufacturer": "ksmosna",
+    "keyboard_name": "k_pro_v1_tkl",
+    "maintainer": "ksmosna",
+    "bootloader": "stm32-dfu",
+    "diode_direction": "COL2ROW",
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true
+    },
+    "matrix_pins": {
+        "cols": ["PA10", "PA9", "PA8", "PC9", "PC8", "PC7", "PC6", "PB15", "PB14", "PB13", "PB12", "PB10", "PB2", "PB1", "PB0", "PC5", "PC4", "PA7"],
+        "rows": ["PB5", "PB4", "PB3", "PD2", "PC12", "PC11"]
```

Comment:
The `P` preceding the pin names needs to be removed. This will not work. Has this firmware actually been tested on hardware?